portsyncd/portsyncd.cpp

```diff
@@ -166,6 +166,12 @@ int main(int argc, char **argv)
         cerr << "Exception \"" << e.what() << "\" had been thrown in deamon" << endl;
         return EXIT_FAILURE;
     }
+    catch (...)
+    {
+        cerr << "Exception had been thrown in deamon" << endl;
```

Comment:
I understand you copied and pasted this message, but "daemon" is misspelled, and the grammar is a bit odd. Could you please correct this spelling in both this message and the one above, and also change "had been" to "was".